### PR TITLE
DAT-13446

### PR DESCRIPTION
**Pull request by sayaliM0412**
## Impact

- [ ] Bug fix (non-breaking change which fixes expected existing functionality)
- [x] Enhancement/New feature (adds functionality without impacting existing logic)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: liquibase-cli/src/main/java/liquibase/integration/commandline/LiquibaseLauncher.java [hunk 1]
```diff
@@ -38,7 +38,7 @@ public static void main(final String[] args) throws Exception {
         File liquibaseHome = new File(liquibaseHomeEnv);
 
         List<URL> urls = new ArrayList<>();
-        urls.add(new File(liquibaseHome, "internal/lib/liquibase-core.jar").toURI().toURL()); //make sure liquibase-core.jar is first in the list
+        urls.add(new File(liquibaseHome, "internal/lib/liquibase-standard.jar").toURI().toURL()); //make sure liquibase-core.jar is first in the list
```
Comment (sayaliM0412) [inline liquibase-cli/src/main/java/liquibase/integration/commandline/LiquibaseLauncher.java:41]: Need to verify this renamed change

File: liquibase-cli/src/main/java/liquibase/integration/commandline/LiquibaseLauncher.java [hunk 2]
```diff
@@ -60,7 +60,7 @@ public static void main(final String[] args) throws Exception {
             }
 
             for (File lib : files) {
-                if (lib.getName().toLowerCase(Locale.US).endsWith(".jar") && !lib.getName().toLowerCase(Locale.US).equals("liquibase-core.jar")) {
+                if (lib.getName().toLowerCase(Locale.US).endsWith(".jar") && !lib.getName().toLowerCase(Locale.US).equals("liquibase-standard.jar")) {
```
Comment (sayaliM0412) [inline liquibase-cli/src/main/java/liquibase/integration/commandline/LiquibaseLauncher.java:63]: Need to verify this renamed change

File: liquibase-dist/src/main/archive/liquibase [hunk 3]
```diff
@@ -53,4 +53,4 @@ fi
 JAVA_OPTS="${JAVA_OPTS-}"
 
 export LIQUIBASE_HOME
-"${JAVA_PATH}" $JAVA_OPTS -jar "$LIQUIBASE_HOME/internal/lib/liquibase-core.jar" ${1+"$@"}
+"${JAVA_PATH}" $JAVA_OPTS -jar "$LIQUIBASE_HOME/internal/lib/liquibase-standard.jar" ${1+"$@"}
```
Comment (sayaliM0412) [inline liquibase-dist/src/main/archive/liquibase:56]: Need to verify this renamed change

File: liquibase-dist/src/main/archive/liquibase.bat [hunk 4]
```diff
@@ -30,4 +30,4 @@ IF NOT DEFINED JAVA_OPTS set JAVA_OPTS=
 set JAVA_PATH=java
 if NOT "%JAVA_HOME%" == "" set JAVA_PATH=%JAVA_HOME%\bin\java
 
-"%JAVA_PATH%" %JAVA_OPTS% -jar "%LIQUIBASE_HOME%\internal\lib\liquibase-core.jar" %*
+"%JAVA_PATH%" %JAVA_OPTS% -jar "%LIQUIBASE_HOME%\internal\lib\liquibase-standard.jar" %*
```
Comment (sayaliM0412) [inline liquibase-dist/src/main/archive/liquibase.bat:33]: Need to verify this renamed change

File: liquibase-integration-tests/setup/build.xml [hunk 5]
```diff
@@ -8,8 +8,8 @@
   <property name="installs.dir" value="${temp.dir}" />
 
   <property name="liquibase.jar.file.dir" value="C:\Tools\liquibase\2.0-123\internal\lib" />
-  <property name="liquibase.jar.file.name" value="liquibase-core.jar" />
-  <property name="liquibase.jar.file.path" value="${liquibase.jar.file.dir}\internal\lib\liquibase-core.jar" />
+  <property name="liquibase.jar.file.name" value="liquibase-standard.jar" />
+  <property name="liquibase.jar.file.path" value="${liquibase.jar.file.dir}\internal\lib\liquibase-standard.jar" />
```
Comment (sayaliM0412) [inline liquibase-integration-tests/setup/build.xml:13]: Need to verify this renamed change

**Comment by nvoxland** [comment]
This looks like it's making a far more invasive change than it should be. I didn't go through all the files, but noticed things like:

- the package we are distributing looks like it's liquibase-standard not liquibase-core now
- the release logic got updated to use liquibase-standard instead of keeping with liquibase-core like it should
- snyk is checking liquibase-standard, not liquibase-core
- the osgi info in liquibase-standard/pom.xml seems to have been changed
- I don't think the liquibase-deployment-pom.xslt files should not have changed
- sub-modules that should not have had their liquibase-core depencencies changed to standard were updated, like liquibase-cdi, liquibase-cdi-jakarta, liquibase-dist, liquibase-maven-plugin
- the cli code is assuming there is a liquibase-standard.jar in the internal/lib dir which there should not be. Still only liquibase-core.jar
- the tar assembly logic has renamed liquibase-core.jar to liquibase-standard.jar
- the "liquibase-core" aggregation logic hasn't been moved out of liquibase-dist to a new liquibase-core sub-module yet
- liquibase-integration-tests should continue to rely on liquibase-core, and has a  bunch of surprising changes to it maybe related to it depending on liquibase-standard now?

File: liquibase-cdi-jakarta/src/main/maven/liquibase-deployment-pom.xslt [hunk 6]
```diff
@@ -12,8 +12,8 @@
     <!-- remove all profiles -->
     <xsl:template match="maven:profiles"/>
 
-    <!-- remove all org.liquibase sub-module dependencies except liquibase-core -->
```
Comment (sayaliM0412) [inline liquibase-cdi-jakarta/src/main/maven/liquibase-deployment-pom.xslt:15]: we want this to still look for `liquibase-core`. So don't change this.

File: liquibase-cdi/pom.xml [hunk 7]
```diff
@@ -15,13 +15,13 @@
     <dependencies>
         <dependency>
             <groupId>org.liquibase</groupId>
-            <artifactId>liquibase-core</artifactId>
```
Comment (sayaliM0412) [inline liquibase-cdi/pom.xml:18]: we want this to still look for liquibase-core. So don't change this.

File: liquibase-cli/pom.xml [hunk 8]
```diff
@@ -15,7 +15,7 @@
     <dependencies>
         <dependency>
             <groupId>org.liquibase</groupId>
-            <artifactId>liquibase-core</artifactId>
```
Comment (sayaliM0412) [inline liquibase-cli/pom.xml:18]: we want this to look for `liquibase-core`. So don't change this.

**Pull request closed without merging**